docs/wave.rst

```diff
@@ -0,0 +1,106 @@
+Acoustic Wave Modelling using Devito
+==================================================
+
+Consider the acoustic wave equation given in 3D:
+
+.. math::
+   m(x,y,z)\frac{\partial^2 u}{\partial t^2} + \eta(x,y,z)\frac{\partial u}{\partial t}-\frac{\partial^2 u}{\partial x^2}-\frac{\partial^2 u}{\partial y^2}-\frac{\partial^2 u}{\partial z^2}= 0
+
+   u(x,y,z,0) = 0
+
+   \frac{\partial u(x,y,z,t)}{\partial t}|_{t=0} = 0
+
+where :math:`\eta` represents damping, :math:`m` represents square slowness
+
```

Comment:
Please detail.
Damp : dampening coefficient for absorbing boundary condition
m  : :math:'\frac{1}{v^2}` where v is the velocity 
u : pressure field